inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/Entrance.java

```diff
@@ -58,7 +58,8 @@ public static void main(String[] args) throws Exception {
         Parser parser;
         if (StringUtils.isEmpty(sqlFile)) {
             final GroupInfo groupInfo = getGroupInfoFromFile(config.getString(Constants.GROUP_INFO_FILE));
-            parser = FlinkSqlParser.getInstance(tableEnv, groupInfo);
+            String auditProxyHosts = config.getString(Constants.METRICS_AUDIT_PROXY_HOSTS);
+            parser = FlinkSqlParser.getInstance(tableEnv, groupInfo, auditProxyHosts);
```

Comment:
I suggest just set `auditProxyHosts` to properties of gruopInfo. not to change FlinkSqlParser.getInstance.

Reply:
ok, thanks. I will resubmit a pr and optimize it.